OSDOCS#8316: Adding removal of LatencySensitive feature set (#66842)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

|===

[id="ocp-4-14-removed-latencysensitive-feature-set"]
==== Support for the LatencySensitive feature set is removed

> As of {product-title} 4.14, support for the `LatencySensitive` feature set has been removed.

Reviewer: ```suggestion
As of {product-title} 4.14, support for the `LatencySensitive` feature set is removed.
```
